milatools/cli/utils.py

```diff
@@ -149,8 +160,12 @@ def __str__(self):
             + "\n\t"
             + "-Retry connecting with mila"
             + "\n\t"
-            + f"-Try to exclude the node with -x {self.node_hostname} "
-            "parameter"
+            + f"-Try to exclude the node with -x {self.node_hostname} parameter\n"
+            + "\n"
+            + "You might want to also include this detailed error message:\n"
+            + "\n```\n"
+            + str(self.error)
+            + "\n```\n"
```

Comment:
```suggestion
            + "If you reach out for help, you might want to also include this detailed error message:\n"
            + "\n```\n"
            + str(self.error)
            + "\n```\n"
```